src/index.html

```diff
@@ -44,10 +44,19 @@ <h1>Instant, themed Twitch chat overlays</h1>
 		allowtransparency="true"
 	>
 	</iframe>
+
+	<template id="output-template">
+		<p>Your URL is:</p>
+		<div>
+			<p class="url"></p>
+			<button name="Copy link to clipboard">Copy</button>
```

Comment:
Howdy! Can you help me understand what you're using the `name` attribute here for?

Reply:
I was trying to recreate the copy button found in codeblocks in the Gatsby docs site: https://www.gatsbyjs.com/docs/quick-start/#getting-started-with-gatsby

Reply:
Truthfully, I'm not sure that `name` is doing anything for them - unless they're using is as their query selector, a la `querySelector('[name="Copy link to clipboard"]')`, or in their styles.

Reply:
I'll remove that part. I assumed (wrongly) it served some accessibility function.